debian/changelog

```diff
@@ -1,6 +1,16 @@
-ubuntu-advantage-tools (19.6) UNRELEASED; urgency=medium
-
-  * New upstream release 
+ubuntu-advantage-tools (19.6) focal; urgency=medium
+
+  * New upstream release. Main changes:
+    - drop SSO interactive login support
+    - d/control: no longer depend on pymacaroons, which was only needed for
```

Comment:
Also could add
- unattached machine will now report availability of services
- dropped dependency on gpg utilities for apt key publication
- command line messages and UI rework

Reply:
`dropped dependency on gpg utilities for apt key publication` was that ever in 19.5.1?

Reply:
I don't think it was

Reply:
What does `command line messages and UI rework` mean? (playing devil's advocate here, i.e., someone who is not familiar with ua reading this). Did the command-line interface change? Will existing script break?

Reply:
Various output messages were changed and status output was modified.

Reply:
good point Andreas, this could break existing scripts if anyone used ua status --format json

Reply:
no one is using the new client yet; this cannot break "existing scripts"